src/Client/webpack.config.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
   devtool: "source-map",
   entry: resolve('./Client.fsproj'),
   output: {
-    path: resolve('./public'),
+    path: resolve('/public'),
```

Comment:
I think this is going to resolve `public` in the root path of your drive (e.g. `C:\public`), you probably want to keep the period or remove the slash.

Reply:
I will keep the `.` then :)